priv/standalone-list/en.json

```diff
@@ -1,9 +1,9 @@
 {
   "sharing": {
-    "caption": "WWW.BOOMBOX.COM",
-    "description_default": "Check out this list.",
-    "hashtags": "list",
-    "title_default": "Check out %{title} on WWW.BOOMBOX.COM",
-    "twitter_title_default": "Check out %{title} on WWW.BOOMBOX.COM %{url}"
+    "list_caption": "WWW.BOOMBOX.COM",
```

Comment:
Having the naming convention like this, and having it in the standalone-list.json is repetitive.  I would add both of these blocks to the standalone.json so we can deprecate these files.